OpenSim/Common/XMLDocument.cpp

```diff
@@ -54,7 +54,8 @@ using namespace std;
 // 30502 for Changing serialization of Geometry
 // 30503 for Changing serialization of Ground
 // 30505 for Changing serialization of Joint to create offset frames
-const int XMLDocument::LatestVersion = 30505;   
+// 30506 for Changing serialization of PrescribedForce to use Connectors
+const int XMLDocument::LatestVersion = 30506;   
```

Comment:
I had this question too... @klshrinidhi and I wanted to make some changes to the code in the 30505 and earlier versions, mainly to fix bugs, but it didn't seem like it would be worthwhile to add a new file version to fix a bug in an unreleased file version.